---
title: LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER | Documents Microsoft
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.prod_service: database-engine
ms.service: ''
ms.component: localdb
ms.reviewer: ''
ms.suite: sql
ms.technology: ''
ms.tgt_pltfrm: ''
ms.topic: reference
ms.assetid: c4288265-43f7-44fd-acab-ba64d04264d9
caps.latest.revision: 10
author: stevestein
ms.author: sstein
manager: craigg
ms.workload: Inactive
ms.openlocfilehash: c6ef82c14f3ff941275e6e0f1e488f53e6300cde
ms.sourcegitcommit: 7a6df3fd5bea9282ecdeffa94d13ea1da6def80a
ms.translationtype: MT
ms.contentlocale: fr-FR
ms.lasthandoff: 04/16/2018
---
# <a name="localdberrorcannotaccessinstancefolder"></a>LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
    
## <a name="details"></a>Détails  
  
|||  
|-|-|  
|Nom du produit|SQL Server|  
|ID d'événement|261|  
|Source de l'événement|Runtime de base de données locale SQL Server 12.0|  
|Composant|API d'exécution de la base de données locale|  
|Texte du message|Impossible d’accéder au dossier de l’instance locale de la base de données : %%LOCALAPPDATA%%\Microsoft\Microsoft DB\Instances locale de SQL Server\\< nom de l’instance\>.|  
  
## <a name="explanation"></a>Explication  
 Un dossier d'instance n'est pas accessible.  
  
## <a name="user-action"></a>Action de l'utilisateur  
  
